components/Starknet/modules/tools/pages/dai_token_migration.adoc

```diff
@@ -0,0 +1,32 @@
+[id="dai_token_migration"]
+= Migrating DAI tokens on Starknet to the new DAI contract
+
+Following link:https://twitter.com/MakerDAO/status/1746977683190251591?s=20
+[Maker DAO’s announcement on Jan 25th], StarkWare launched a new DAI contract and bridge on Starknet.
+
+The new DAI token and bridge are a part of StarkGate and compatible with StarkGate 2.0 features.
+
+You can withdraw old DAI tokens (DAI v0) without any limitation. Depositing using the DAI v0 bridge are disabled. You are encouraged to migrate to the new DAI token. You can use swap services on Starknet to swap DAI v0 for DAI.
+
+You are encouraged to migrate your old DAI (DAI v0) tokens for new DAI tokens.
+
+Maker DAO's DAI token on Starknet is written in Cairo0 and is not upgradeable. Without upgradability, it cannot support StarkGate's latest features, such as Smart Deposits and Withdrawal Limits, and over time it will stop being compatible with Starknet altogether (Regenesis). This means that a transition plan is necessary.
+
+On January 25th, StarkWare launched a new set of DAI bridge and token contracts under StarkGate, written in Cairo. This new DAI token will retain the same contract ’symbol’ and ’name’ as the existing one. To differentiate between the two on Apps and other UIs, we refer to the old DAI as “DAI v0” and the new DAI simply as “DAI.”
+
+
+== Migrate from v0 Starknet DAI tokens to new Starknet DAI tokens
```

Comment:
Ok, but I think it would be better to be as specific as possible. The naming you chose might be appropriate for conversations and targeted communications, but this page will show up in search results, and it'll probably generate false positives if we don't specify that it's referring to Starknet. I'll add more details in the :description: metadata.